deploy/kubernetes/overlays/prow-head-template/controller_images.yaml

```diff
@@ -0,0 +1,10 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-controller
+spec:
+  template:
+    spec:
+      containers:
+        - name: gce-pd-driver
```

Comment:
TODO: once pull job is passing, modify this to use the gke-release-staging repository

Reply:
turns out release-staging is not public, so we can't use it for oss testing.